### DISCUSSION TITLE
Notification Panel On Home Screen

### DISCUSSION THREAD

**Discussion by cybozax** [MM2-0/Kvaesitso#723] [Ideas]
I love the idea of scrolling for widgets and music weather etc. and was thinking 'can it be implemented for notification as well'?

Can we have a notifications for select apps on the scrollable home screen as sliding down for notifications is a bit clunky on launchers except the default that came with device. 

Ability to reply to text, quick reply and maybe copy otp like the notify panel.

**Comment by kartik2005221** [comment]
great idea 👍
